cmd/commander/commander.go

```diff
@@ -63,9 +63,20 @@ func createTestCommand() cli.Command {
 				Name:  "dir",
 				Usage: "Execute all test files in a directory sorted by file name, this is not recursive - e.g. /path/to/test_files/",
 			},
+			cli.StringFlag{
```

Comment:
I think I mentioned this previously. But any thoughts on supporting multiple filter flags? I believe you would just need something like
```golang
cli.StringSliceFlag{
    Name: "filter",
},
```
And then loop in `app.go`.

Reply:
Good point!